packages/theme/src/theme-provider.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, PropsWithChildren } from 'react'
+import styled, { ThemeProvider as StyledThemeProvider } from 'styled-components'
+import { reverseThemeMap, themeDark, themeDefault, themeLight } from './themes'
+import { initColors } from './document-head-contents'
+import { generateCssColorVariables } from './utils/generate-css-color-variables'
+import { Theme } from './types'
+
+// you need this if you're using themes, so we initiate this automatically
+initColors()
```

Comment:
Why do we call it here and in cookied provider?

Reply:
if we're using any of providers this should be ran before react. will fix